setup.cfg

```diff
@@ -19,12 +19,4 @@ env =
 	TEST=True
 	MODE=PRODUCTION
 	DJANGO_SETTINGS_MODULE=config.settings.production
-addopts = --cov=mathesar --cov=db --cov-config=setup.cfg --maxprocesses=1 -n auto
-
-# The below specifies some pytest-cov options
```

Comment:
Is there a reason for removing the omitted directories?

Reply:
Actually, this reminds me of something. I want to try adding the coverage report back in.

Reply:
And now I remember why I took it out. Pytest reruns all tests when you have the coverage reporting turned off.

The reason for removing the omitted directories is that they didn't seem to make sense since we aren't doing coverage reporting anyway.

Reply:
Cool, I approved the changes. Please feel free to merge it.